tests/test_1_location_to_square.py

```diff
@@ -0,0 +1,38 @@
+from src.qth_locator import location_to_square
+
+
+def test_location_to_square_positive():
+    squares = [
+        (None, (0, 0)),
+        (None, (32, 26)),
+        ("KM32jn", (32.541667, 26.75)),
+        ("KM32jn", (32.570831, 26.750003)),
+        ("KM72kk", (32.437487, 34.875015)),
+        ("JN45fo", (45.603333, 8.456667)),
+        ("KO92so", (52.601484, 39.565160)),
+        ("KM72jb", (32.071209, 34.780089)),
+        ("KM72jb", (32.07, 34.78)),
+    ]
```

Comment:
Why there is a need for 9 square cases?

I would recommend to build 1 important happy case, or several important ones when conditions are changing, and then around edge cases.

Reply:
Yes, we want more locations to test, because of the nature of how they are counted.
1 Square can hold many locations (lat/long combinations) basically square is not an exact location, It is a map square with specific width and height. In another hand, one location should be in one specific square.